### ISSUE TITLE
ansible-playbook aws-setup.yaml not working

### ISSUE THREAD

**Issue by jcaberio** [book-of-kubernetes/examples#27]
I'm trying out the examples on a new machine. I've already done the setup. When I run `ansible-playbook aws-setup.yaml` 
I keep on getting this error no matter how many times I retry

![Screenshot 2023-05-20 at 5 02 58 PM](https://github.com/book-of-kubernetes/examples/assets/21195237/27c36887-84be-4caf-b5a4-664d205b1cb4)

```
TASK [aws-instances : instances] **************************************************************************************************************************************************************
fatal: [localhost]: FAILED! => {"msg": "The task includes an option with an undefined variable. The error was: list object has no element 0. list object has no element 0\n\nThe error appears to be in '/Users/jorickcaberio/code/book-of-kubernetes/examples/setup/roles/aws-instances/tasks/setup.yaml': line 93, column 3, but may\nbe elsewhere in the file depending on the exact syntax problem.\n\nThe offending line appears to be:\n\n\n- name: instances\n  ^ here\n"}
```

I read in the setup folder README that

> Note that creation will take a while as we need to wait for the instance to finish initializing before we can collect the SSH host key.
> 
> NOTE Ansible will warn that the instance did not start before timeout. This is expected and the task to fetch host keys has a retry in it to wait for instance startup. Also note that if AWS is a little slow in assigning a public IP to an instance, Ansible may fail the first time. This is not an issue; just run it again and it should succeed.

How long should I wait? I don't see any instances in my AWS console initializing. I tried waiting for several minutes before rerunning the command but it doesn't work.

**Comment by jcaberio** [comment]
Solved by https://github.com/book-of-kubernetes/examples/pull/23
